test/framework/src/main/java/org/elasticsearch/search/aggregations/AggregatorTestCase.java

```diff
@@ -590,6 +590,7 @@ private <A extends InternalAggregation, C extends Aggregator> A searchAndReduce(
         try {
             if (randomBoolean() && aggs.size() > 1) {
                 // sometimes do an incremental reduce
+                logger.info("Doing incremental reduce");
```

Comment:
Do we need this line? Perhaps it was there for temporary debugging?

Reply:
This was added in this PR. Maybe to debug something. I think we should not add it?

Reply:
I removed it.